src/components/Pagination/Pagination.stories.mdx

```diff
@@ -110,13 +114,60 @@ const onPageChange = (currentPage: number) => {
     name="Pagination with Unchanging URL"
     args={{
       className: undefined,
-      getPageHref: undefined,
       id: "pagination-id-2",
       initialPage: 7,
-      onPageChange: undefined,
+      onPageChange: (selectedPage) => {
+        console.log(`Current page: ${selectedPage}`);
+      },
+      pageCount: 100,
+    }}
+  >
+    {(args) => <Pagination {...args} />}
+  </Story>
+</Canvas>
+
+### Forcing a Page Change Programatically
+
+Generally, the selected page will change when a user clicks on a page number or
+the `Previous`/`Next` buttons in the component. However, there may be circumstances
+when you want to programatically change the page without the user explicitly
+requesting it (for example, if you want a user to be brought back to page 1 after
+entering a new search term). In this case, you can use the `currentPage` prop to
+force this change. Note that the `currentPage` prop only works with the unchanging
+URL `Pagination`.
+
+The following example shows that the user can be brought to page 1 without clicking
+on the `Pagination` component.
+
+<Canvas withToolbar>
```

Comment:
This is more of a further example and explanation of the previous Story, so I don't think it needs this Story here. We also don't want to have Controls for this since it's not a variant. You can swap the `Story` wrapper for a `DSProvider` wrapper and not have the `args`; it's fine to hardcode the props for this example.

```
<Canvas>
  <DSProvider>
    {() => {
      const [page, ...
      // ...
    }}
  </DSProvider>
</Canvas>

You can also make the internal `{() => {....}}` into a component if you want.
```